Neos.Fusion/Classes/Core/Cache/ParserCache.php

```diff
@@ -105,4 +108,29 @@ private function getAbsolutePathForPackageRessourceUri(string $requestedPath): s
         $package = $this->packageManager->getPackage($resourceUriParts['host']);
         return Files::concatenatePaths([$package->getResourcesPath(), $resourceUriParts['path']]);
     }
+
+    /**
+     * Uses the same technique to resolve a package resource URI like Flow.
+     *
+     * package://My.Site/NodeTypes/Foo/Bar.fusion
+     * ->
+     * FLOW_PATH_ROOT/Packages/Sites/My.Package/NodeTypes/Foo/Bar.fusion
+     *
+     * {@see \Neos\Flow\ResourceManagement\Streams\ResourceStreamWrapper::evaluateResourcePath()}
+     * {@link https://github.com/neos/flow-development-collection/issues/2687}
+     *
+     * @throws \InvalidArgumentException
+     */
+    private function getAbsolutePathForNodetypesUri(string $requestedPath): string
```

Comment:
Nitpick: "..NodeTypes" is camel-cased in other places IIRC
```suggestion
    private function getAbsolutePathForNodeTypesUri(string $requestedPath): string
```